components/Button.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 14px 52px;
+  gap: 12px;
+  border: none;
+  border-radius: 55px;
+  width: 100%;
+  color: var(--white);
+  background: var(
+    ${({ isPrimary }) =>
+      isPrimary ? "--primary-gradient" : "--secondary-gradient"}
+  );
+
+  &:hover {
+    background: var(
+      ${({ isPrimary }) => (isPrimary ? "--primary" : "--secondary")}
+    );
+  }
+  &:disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
```

Comment:
cursor:not-allowed ? sounds interesting.. am I not allowed to hover it 😅

Reply:
No. Your only allowed to click on the button, if the user has entered a name. But good info for the hover. I'll remove the hover effect from disabled btn